core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionMonitorActor.scala

```diff
@@ -110,7 +172,9 @@ trait SubmissionMonitor extends FutureSupport with LazyLogging {
 
     def abortQueuedWorkflows(submissionId: UUID) = {
       datasource.inTransaction { dataAccess =>
-        dataAccess.workflowQuery.batchUpdateWorkflowsOfStatus(submissionId, WorkflowStatuses.Queued, WorkflowStatuses.Aborted)
+        workflowStatusCounter(WorkflowStatuses.Aborted).countDBResult {
```

Comment:
My thinking was that `WorkflowComponent` should only be concerned with updating the database, and that metric updates should be at the level above it (in the actual business logic). Plus, these metrics usually need a `Workspace` and a `Submission` which would need to be passed into `WorkflowComponent` solely for the purpose of updating metrics, which doesn't seem right.

Maybe instrumentation relating specifically to the database could live at the `*Component` layer (such as query timing, etc).